.gitlab-ci.yml

```diff
@@ -19,7 +19,7 @@ check_dependencies:
     - tags
 
 .build_rom_apu: &build_rom_apu
-  image: coreboot/coreboot-sdk:1.52
+  image: coreboot/coreboot-sdk:2021-04-06_7014f8258e
```

Comment:
I guess we have different tag?

Reply:
we don't use the pce-fw-builder container in CI, we use upstream coreboot-sdk as the container. I'm not sure why we do this, maybe we could change it to pce-fw-builder instead?